src/scss/components/switch/_index.scss

```diff
@@ -0,0 +1,178 @@
+@import "./config";
+
+hx-switch-control {
```

Comment:
#### Please manually test the Switch component in - Chrome, Firefox, Safari, Legacy Edge, and IE11.

Reply:
Yes, i have tested manually in all browsers it looks fine for me. There are no breaks in stylings.